Lib/defcon/objects/layoutEngine.py

```diff
@@ -38,15 +38,15 @@ def _layoutEngineOTLTablesRepresentationFactory(layoutEngine):
             addOpenTypeFeatures(feaPath, otf)
         except:
             import traceback
-            print traceback.format_exc(5)
+            print(traceback.format_exc(5))
         finally:
-            # XXX clean up file on disk
+            os.close(fd)
             os.remove(feaPath)
-        if otf.has_key("GDEF"):
```

Comment:
Nice! I didn't know that TTFont had __contains__ now. I've been wanting that for a loooooong time.